hack/local-up-volcano.sh

```diff
@@ -94,9 +95,13 @@ prepare
 
 if [ "${INSTALL_MODE}" == "kind" ]; then
   kind-up-cluster
-  export KUBECONFIG="$(kind get kubeconfig-path ${CLUSTER_CONTEXT})"
+  export KUBECONFIG="$(kind get kubeconfig ${CLUSTER_CONTEXT})"
 fi
 
+kubectl delete deploy volcano-admission -n $NAMESPACE
```

Comment:
l don't think it's a good idea to hard code volcano here, due to the reason that we may deploy volcano using helm with some other prefix.

In addition, if we set `ImagePullPolicy` to `Always`, the latest build image will be pulled every time you run this script.

Reply:
We've set daily job to build the latest image and push to dockerhub, so that `ImagePullPolicy` set to `Always` can ensure you get the latest version

Reply:
No, i think local-up-volcano.sh script just use for develop.
 I just want to execute this script every time I finish developing the code, it will automatically update the mirror I just developed for debugging.

Reply:
And helm can’t know if I have updated, because the name of the image created is the same every time

Reply:
Sorry for it cannot be satified now. You can build image locally and load it to your develop environment. Replace the deployment yaml `image` field to debug.

Reply:
FYI: generally my way to do that is to deploy all the component with `ImagePullPolicy: Always` and delete all the **related pods**, then all the component will be updated.

Reply:
Even if you want to debug using this script, is `uninstall-volcano` before `install-volcano` work?

Reply:
I’m ok.